src/hyp3_isce2/insar_stripmap.py

```diff
@@ -24,31 +32,86 @@
     os.environ['PATH'] = str(ISCE_APPLICATIONS) + os.pathsep + os.environ['PATH']
 
 
-def insar_stripmap(reference_scene: str, secondary_scene: str) -> Path:
+# TODO update docstring description
+def insar_stripmap(user: str, password: str, reference_scene: str, secondary_scene: str) -> Path:
     """Create an interferogram
 
     This is a placeholder function. It will be replaced with your actual scientific workflow.
 
     Args:
+        user: Earthdata username
+        password: Earthdata password
         reference_scene: Reference scene name
         secondary_scene: Secondary scene name
 
     Returns:
         Path to the output files
     """
+    session = asf.ASFSession().auth_with_creds(user, password)
+
+    results = asf.granule_search([reference_scene, secondary_scene])
+
+    polys = []
+    durls = []
+    for result in results:
+        if 'L1.0' in result.properties['url']:
+            polys.append(Polygon(results[0].geometry['coordinates'][0]))
+            durls.append(result.properties['url'])
```

Comment:
We no longer need to filter to `L1.0` scenes manually. Also, should first line reference `result` rather than `results[0]`?
```suggestion
    # TODO should this reference `result` rather than `results[0]`?
    polys.append(Polygon(results[0].geometry['coordinates'][0]))
    durls.append(result.properties['url'])
```

Reply:
Yes, you're right it should be 'result'. Thanks for catching the error